agent/engine/docker_task_engine.go

```diff
@@ -945,6 +959,24 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	return metadata
 }
 
+// Log driver options type, tag and fluentd-address are added
+// For reference - https://docs.docker.com/config/containers/logging/fluentd/
+func getLogRouterConfig(task *apitask.Task, container *apicontainer.Container, hostConfig *container.HostConfig, engine *DockerTaskEngine) container.LogConfig {
+	seelog.Infof("Since the value for logrouter in container is %s, we modify LogConfig values in hostconfig to support log router driver", logConfigType)
```

Comment:
I wouldn't put this on the getter. I'd move it to where it gets applied. Consider setting it as a debug level log.

Also, this is super friendly and helpful, but for the logging we should be more terse. Something like:
`"applying logrouter config for container %s"`

Remember that the consumers of the log will be us or other devs - folks that'll be able to search the for the error string to find whats going on if they suspect problems.